benchmark/largesetresult/largesetresult_test.go

```diff
@@ -28,6 +29,39 @@ func BenchmarkLargeResultSet(*testing.B) {
 	runLargeResultSet()
 }
 
+// getDSN constructs a DSN based on the test connection parameters
+func getDSN() (string, *sf.Config, error) {
```

Comment:
Nit: I think you might want to rename check to failOnMissing or something along those lines

Reply:
Agree.

Reply:
sorry I took something different. Can you elaborate your comment? Are you suggesting changing the method name?

Reply:
ah, ok. you were referring to `env`.